src/styles/quiz-card/quiz-card.html

```diff
@@ -0,0 +1,15 @@
+<section class="quiz-card">
+  <h1 class="quiz-card__headline">Questions</h1>
```

Comment:
This shouldn't be a `h1` heading as you most likely will have this in your header and the next heading would be a `h2`
```suggestion
  <h2 class="quiz-card__headline">Questions</h2>
```